### PR TITLE
Translate scope to Japanese.

### PR DESCRIPTION
**Pull request by ledsun**

### CODE REVIEW

File: problems/scope/problem_ja.md [hunk 1]
```diff
@@ -1,5 +1,76 @@
...
+# スコープ
 
+「スコープ」は参照できる変数・オブジェクト・関数の集合です。
+
+JavaScriptには、二つのスコープがあります。グローバルとローカルです。
+関数定義の外側で定義した変数はグローバルスコープです。プログラムのどこからでも参照して変更することができます。
+関数定義の内側で定義した変数はローカルスコープです。関数が実行されるときに作られ、関数が終了すると破棄されます。
+関数外のプログラムからは参照できません。
+
+他の関数の中で定義した関数を「ネストした関数」と呼びます。「ネストした関数」からは親関数のスコープを参照できます。
```
Comment (marocchino) [inline problems/scope/problem_ja.md]: How about "入れ子関数"?

File: problems/scope/problem_ja.md [hunk 2]
```diff
@@ -1,5 +1,76 @@
...
+		console.log( a, b, c );
+	}
+
+	bar(b * 4);
+}
+
+foo(); // 4, 2, 48
+```
+
+即時実行関数式 (Immediately Invoked Function Expression : IIFE) という共通パターンで、ローカルスコープを作れます。
```
Comment (marocchino) [inline problems/scope/problem_ja.md]: I think common will be "一番的な" rather than "共通".

Comment (marocchino) [inline problems/scope/problem_ja.md]: but,it's fine in case of "共通パターン" is more commonly used in Japanese.

**Comment by marocchino** [comment]
@ledsun any update?

**Comment by ledsun** [comment]
I cannot decide proposals are better or not.

**Comment by marocchino** [comment]
Actually, Original translation is good enough. Sorry for delay this. :)

**Pull request merged**